Traceback (most recent call last):
  File "/media/lab/data/yzk/ManiSkill-ViTac2024/scripts/train_sac.py", line 179, in <module>
    model.learn(
  File "/media/lab/data/yzk/ManiSkill-ViTac2024/scripts/../solutions/sac_custom.py", line 307, in learn
    return super().learn(
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/site-packages/stable_baselines3/common/off_policy_algorithm.py", line 301, in learn
    total_timesteps, callback = self._setup_learn(
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/site-packages/stable_baselines3/common/off_policy_algorithm.py", line 284, in _setup_learn
    return super()._setup_learn(
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/site-packages/stable_baselines3/common/base_class.py", line 424, in _setup_learn
    self._last_obs = self.env.reset()  # type: ignore[assignment]
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/site-packages/stable_baselines3/common/vec_env/subproc_vec_env.py", line 138, in reset
    results = [remote.recv() for remote in self.remotes]
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/site-packages/stable_baselines3/common/vec_env/subproc_vec_env.py", line 138, in <listcomp>
    results = [remote.recv() for remote in self.remotes]
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/multiprocessing/connection.py", line 255, in recv
    buf = self._recv_bytes()
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/multiprocessing/connection.py", line 419, in _recv_bytes
    buf = self._recv(4)
  File "/home/lab/anaconda3/envs/mani_vitac/lib/python3.10/multiprocessing/connection.py", line 388, in _recv
    raise EOFError
EOFError